test/lib/fixtures/resolutions/yarn2/expected-tree.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "adm-zip": {
+      "labels": {
+        "scope": "prod",
+        "missingLockFileEntry": "true"
+      },
+      "name": "adm-zip",
+      "version": "0.4.8"
```

Comment:
I have changed the tests on this line to reflect expected behaviour - that the package version in `resolutions` is what snyk picks up. 

This is because the resolution package will be the version the user will be using within their code base so makes most sense to make sure we scan it.

Reply:
I agree that is the correct behavior, but it breaks the test. I wasn't attempting to fully solve the `resolutions` use case in the original PR. I was just hoping to unblock Yarn 2 users.

Reply:
Hey @mhassan1, sorry for not being clearer, we tested your proposal against this test case to highlight what we expected based on what we understood from the docs ([resolutions](https://classic.yarnpkg.com/en/docs/package-json/#toc-resolutions)) and the behaviour of snyk. 

We expected that if we scan a project with resolutions snyk will scan the resolutions versions for vulnerabilities , but we see that it scans the version in the dependencies instead which could lead to missed vulnerabilities which is a problem for us.

We don’t have capacity to figure this out this sprint, so if you can come up with a solution sooner than we can get to this, we would highly appreciate it. Moreover, if our understanding is wrong please feel free to correct us.

Reply:
I understand that missed vulnerabilities due to `resolutions` is a problem for Snyk, but this PR was not meant to fix that issue. The `resolutions` problem has always been unhandled with both Yarn 1 and Yarn 2 in the past. This PR was only to unblock Yarn 2 users (who right now can't use the latest version of the Snyk CLI at all if they have any `resolutions`). I think a working Snyk CLI that doesn't correctly consider `resolutions` is better than a broken Snyk CLI, but feel free to disagree.

I'm happy to help figure out the `resolutions` problem, but that may be a harder problem to get right, since the logic for `resolutions` may be complicated. I think that conversation is better suited to a new PR.